test/e2e/e2e.go

```diff
@@ -440,6 +440,16 @@ func runCommand(cmd ...string) (string, error) {
 	return string(output), nil
 }
 
+// mustRunCommand runs the cmd and returns the combined stdout and stderr, after asserting
+// the command didn't raise any error
+func mustRunCommand(cmd ...string) (string, error) {
```

Comment:
Is this a left-over that needs to be removed?

Reply:
yeah I had the same comment and after that I am LGTM

Reply:
yes, cleaning...